requirements.txt

```diff
@@ -0,0 +1,5 @@
+streamlit==1.31.1
+manim==0.18.0
+numpy==1.26.4
+matplotlib==3.8.3
```

Comment:
you do not need to add this file to git please use the command as above to untrack it from git and keep it local to your computer only